README.md

```diff
@@ -65,7 +65,7 @@ Adapt the page to the following screens:
 11. `git push origin develop` - to send you code for PR.
 12. Create a Pull Request (PR) from your branch `develop` to branch `master` of original repo.
 13. Replace `<your_account>` with your Github username in the
-  [DEMO LINK](https://<your_account>.github.io/layout_dia/).
+  [DEMO LINK](https://Dimak123.github.io/layout_dia/).
```

Comment:
<img width="736" alt="image" src="https://user-images.githubusercontent.com/38065883/167921625-7a3e2e5f-f07f-49d0-8151-1b94a06af7fb.png">
Wrong color

Reply:
Only arrow button should be scaled when card is hovered(check design)
<img width="859" alt="image" src="https://user-images.githubusercontent.com/38065883/167921753-87d50a68-3708-4fa0-948e-078020ea1d8a.png">

Reply:
+

Reply:
Dome

Reply:
Done

Reply:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/38065883/168346893-e7edbc44-0cd8-4f00-895b-ffccb577a439.png">
Still wrong color here

Reply:
https://user-images.githubusercontent.com/38065883/168346960-53fcca7e-c212-44b2-bc12-27f05445813b.mov

Button should be scaled when I hover card, but not button itself